Country.py

```diff
@@ -0,0 +1,270 @@
+import os
+import pygame
+import random
+
+BLACK = (0, 0, 0)
+RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+BLUE = (0, 0, 255)
+
+
+pygame.init()
+font = pygame.font.SysFont('Consolas', 20)
+
+class Country:
+    """
+    Each country has its maximum population without any infected people,
+    before we click the country in the beginning of the game.
+    """
+    def __init__(self, x, y, max_pop, radius=50, color=RED, infected_pop=0, infected_rate=1.1, dead_pop=0, death_rate=1, airborne_rate=0):
+        """
+        Position and color of each country are defined.
+        Population(Infected, death, toal), and Rate(Infection, death, airborne) are defined.
+        """
+        self.initial_pos = (x, y)
+        self.x = x
+        self.y = y
+        self.color = color
+        self.radius = radius
+        self.infected_pop = infected_pop
+        self.infected_rate = infected_rate
+        self.max_pop = max_pop
+        self.dead_pop = dead_pop
+        self.death_rate = death_rate
+        self.airborne_rate = airborne_rate
+
+    def infected_ratio(self):
+        """
+        The ratio of infection (population of infected people / toal population of a country)
+        """
+        if self.max_pop != 0:
+            return int(self.infected_pop) / self.max_pop
+        else:
+            return 1
+
+    def death(self):
+        """
+        A part of the infected population would be passed away.
+        Then the infected population and maximum population will be reduced as many as the number of people death.
+        """
+        death_pop = 0
+        alive_pop = self.max_pop
+        if self.infected_ratio() > 0.10:
+            if self.infected_pop > 10:
+                death_pop = int(self.death_rate*self.infected_pop*(random.random()/15))
+            else:
+                if self.max_pop >= 1:
+                    death_pop = 1
+                else:
+                    death_pop = 0
+
+        self.infected_pop = self.infected_pop - death_pop
+        self.max_pop -= death_pop
+        self.dead_pop += death_pop
+
+    def step(self):
+        """
+        If infection starts (infect_pop changed into unity),
+        then the infection starts with the certain rate
+        """
+        if self.infected_pop < self.max_pop:
+            self.infected_pop = self.infected_pop * self.infected_rate
+            if self.infected_pop >= self.max_pop:
+                self.infected_pop = self.max_pop
+
+        """return integer part of infected population"""
+        # return int(self.infected_pop)
+
+        """return probability"""
```

Comment:
For comments that are inside a function and describe the inside view *how* some *lines of code* work (as opposed to docstrings, that are at the top and describe the outside view of *what* the *whole function* does), use `# comments` instead of `"""doc strings"""`.